bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -150,7 +150,11 @@ def _bytes_to_json(value):
 def _timestamp_to_json(value):
```

Comment:
I know this is only used in BigQuery right now, but since it's kind of BigQuery specific, should the method name be something like `_bq_timestamp_to_json`?